cluster/examples/kubernetes/minio/operator.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rook-minio-system
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: objectstores.minio.rook.io
+spec:
+  group: minio.rook.io
+  names:
+    kind: ObjectStore
+    listKind: ObjectStoreList
+    plural: objectstores
+    singular: objectstore
+  scope: Namespaced
+  version: v1alpha1
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: rook-minio-operator
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  - secrets
+  - pods
+  - services
+  - nodes
+  verbs:
+  - get
+  - watch
+  - create
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  verbs:
+  - get
+  - create
+- apiGroups:
+  - extensions
+  resources:
+  - deployments
+  verbs:
+  - get
+  - create
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterroles
```

Comment:
@tonya11en where is the minio operator creating RBAC objects such as clusterroles and clusterrolebindings?  where does it create deployments? I'm not sure these permissions are needed either.  What's the process you're following to get the minimal set of permissions?